summaries/summary-default.toml

```diff
@@ -0,0 +1,3240 @@
+# Summary for Libra subset 'default'. @generated by x.
+# To regenerate, run 'cargo x generate-summaries'.
```

Comment:
I don't think it is normal that your PR is adding this oO

Reply:
Yeah I have no idea. I followed some instructions and was told to generate this using a script... But it's strange. I'll try removing it and see if it fails or not.